### PR TITLE
Unit tests for action commands

### PR DESCRIPTION
**Pull request by ajpal**
Unit test coverage for `changePropBy`, `jumpTo`, `moveForward`, `moveInDirection`, `moveToward`, `setProp`, and `turn`

### CODE REVIEW

**Review by epeach — COMMENTED** [review]
PR Description?

File: apps/test/unit/gamelab/spritelab/actionCommandsTest.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+  it('changePropBy', () => {
+    let spriteId = makeSprite(undefined, {x: 123, y: 321});
+    commands.changePropBy(spriteId, 'x', 100);
+    expect(spriteCommands.getProp(spriteId, 'x')).to.equal(223);
+
+    commands.changePropBy(spriteId, 'y', 100);
+    expect(spriteCommands.getProp(spriteId, 'y')).to.equal(400 - 221);
+
+    commands.changePropBy(spriteId, 'direction', 200);
```
Comment (epeach) [inline apps/test/unit/gamelab/spritelab/actionCommandsTest.js:23]: Can we confirm that the direction starts at 0 to show that the initial changePropBy('direction', 200) changes the value by the correct amount?

File: apps/test/unit/gamelab/spritelab/actionCommandsTest.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      commands.moveForward(spriteId, 100);
+      expect(spriteCommands.getProp(spriteId, 'x')).to.equal(100);
+      expect(spriteCommands.getProp(spriteId, 'y')).to.equal(400 - 0);
+    });
+
+    it('can move vertically', () => {
+      let spriteId = makeSprite(undefined, {x: 0, y: 0});
+      commands.setProp(spriteId, 'direction', 90);
+      commands.moveForward(spriteId, 100);
+      expect(spriteCommands.getProp(spriteId, 'x')).to.be.within(0, 0.1);
```
Comment (epeach) [inline apps/test/unit/gamelab/spritelab/actionCommandsTest.js:49]: Is there a reason why this isn't `to.equal`?

File: apps/test/unit/gamelab/spritelab/actionCommandsTest.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      spriteId = makeSprite(undefined, {x: 0, y: 0});
+    });
+    it('sets direction', () => {
+      commands.setProp(spriteId, 'direction', 100);
+      expect(spriteCommands.getProp(spriteId, 'direction')).to.equal(100);
+      commands.setProp(spriteId, 'direction', 400);
+      expect(spriteCommands.getProp(spriteId, 'direction')).to.equal(40);
+    });
+
+    it('sets y', () => {
```
Comment (epeach) [inline apps/test/unit/gamelab/spritelab/actionCommandsTest.js]: Nit: I'm not sure this set is necessary.

**Review by epeach — COMMENTED** [review]
Just a few nits.

File: apps/test/unit/gamelab/spritelab/actionCommandsTest.js (hunk 2 above)
Comment (ajpal) [reply apps/test/unit/gamelab/spritelab/actionCommandsTest.js:49]: Just floating point imprecision
image

**Review by epeach — APPROVED** [review]
LGTM

**Pull request merged**